userguide/virtualmachines.rst

```diff
@@ -174,14 +201,14 @@ Virtual machine configuration options are described in
    +----------+--------------------+----------------+-----------------------------------------------------------------------------------------------+
    | 4        | Disk image         | check option   | Select :guilabel:`Create new disk image` to create a new zvol on an existing dataset.         |
    |          |                    | with custom    | This is used as a virtual hard drive for the VM. Select :guilabel:`Use existing disk image`   |
-   |          |                    | fields         | to :guilabel:`Browse` to an existing zvol or file for the VM.                                 |
+   |          |                    | fields         | and click |ui-browse| to select an existing zvol or file for the VM.                          |
```

Comment:
I think we determined that files are not allowed, only zvols.  Please verify.

Reply:
testing with latest 11.2 only displays created zvols in the drop-down menu. Updating text